mpas_analysis/config.default

```diff
@@ -596,6 +596,24 @@ movingAveragePoints = 12
 # yearStrideXTicks = 1
 
 
+[timeSeriesSSHAnomaly]
+## options related to plotting time series of sea surface height (SSH)
+## anomalies from a reference year
+
+# Number of points over which to compute moving average(e.g., for monthly
+# output, movingAveragePoints=12 corresponds to a 12-month moving average
+# window)
+movingAveragePoints = 1
```

Comment:
```suggestion
movingAveragePoints = 12
```
@golaz and @maltrud, based on the updated plot I just included, I think we probably want to average out the annual cycle.  Would you agree?

Reply:
I think we do want to remove the seasonal cycle.

Reply:
![image](https://user-images.githubusercontent.com/4179064/119832063-2401cb00-befe-11eb-99ab-fe14e6a86d4a.png)

Reply:
that looks great!